index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+
+    <title>Software Developer - Portfolio</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="navcontainer">
+        <nav class="navbar">
+          <div class="logo">My Logo</div>
+          <div class="hamburger">
+            <i class="fa fa-bars"></i>
+          </div>
+
+          <ul class="menu menu-list">
+            <li>Portfolio</li>
+            <li>About</li>
+            <li>Contact</li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <section class="section">
+      <div class="sectionheading">
+        <p>I'm Collins.</p>
+        <p>Glad to see you!</p>
+      </div>
+
+      <div class="sectionparagraph">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hesitate to contact me.
+      </div>
+      <span class="socialmedia">Lets Connect</span>
+      <ul class="media">
+        <li><i class="fa-brands fa-twitter"></i></li>
+        <li><i class="fa-brands fa-linkedin-in"></i></li>
+        <li><i class="fa-brands fa-github"></i></li>
+        <li><i class="fa-brands fa-facebook-f"></i></li>
+      </ul>
```

Comment:
- [ ] Please ensure that your icon color matches the given design as part of the requirement that says: `Your website should look exactly like the design (e.g. font, colors, images, text, margins)` using this [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&mode=design&t=KmxbePoLhAgnW9QY-0)

Below is the screenshot showing the difference. :arrow_down: 

<table>
    <tr>
      <td>Your design </td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

 
![student Icons](https://github.com/enocol/MobileFirst/assets/43528594/ba25f56b-5255-4322-8a0c-375eed485848)


</td>
      <td>

![Figma Icons](https://github.com/enocol/MobileFirst/assets/43528594/84f97cc5-82bc-4319-9a7c-b8bc3f5a69fd)


</td>
    </tr>
  </table>